Annotation/declarative_golden_set.json

```diff
@@ -62,7 +62,7 @@
                     }
                 ],
                 "type": "declarative",
-                "clause": "",
+                "clause": "that that is precisely what happened on this day",
                 "embedded clauses": []

```

Comment:
Arguably a nested EC with "what happened on this day"

Reply:
I think that would be the complement of the copula, and therefore we should disregard it